doc/modules/ROOT/pages/tutorials/node-regression-with-subgraph-and-graph-sample.adoc

```diff
@@ -1,6 +1,6 @@
 // DO NOT EDIT - AsciiDoc file generated automatically
 
-= Node Regression with Subgraph and Graph Sample projections
+= Node Regression, (Sub)Graph Sample
```

Comment:
This one becomes a bit brief IMO -- just two names with a comma between.
```suggestion
= ML pipelines: Node Regression, Subgraph & Sampling
```

That's still 51 chars

Reply:
No, the thing is, the limit is 70 including the name of the page, which is "Neo4j Graph Data Science Client" 😅.. but if making it this short renders it wrong, then we can keep it. It's not that bad for performance

Reply:
now the title gets close to a list of keywords 😅

Reply:
Is there a way to somehow make the SEO read the page title as `Neo4j GDS Python Client` instead? Without changing the displayed title on the page.

Reply:
Yeah not all of my suggestions here are great. View them as experimental suggestions that would fit within the limit. I am not convinced that all of them should be used.

Reply:
@Mats-SX Here too, if we lose the specific ML task (regression) we could do "ML on nodes: Subgraphs and sampling". If this is not descriptive enough, let's leave it as is.

Reply:
I say leave it as-is in this case then